quizzes/07_parameters.test.js

```diff
@@ -0,0 +1,50 @@
+console.log(
+  // randGen();
+  // weatherOnDemand(arr, 'Bali', 'London')``
+)
+
+function randGen(n) {
+  // generates a random integer betweeb 0 and n
+  // assume n is always positive
+  // randGen(n) can be at least 0 and at max n - 1
+  // If n is undefined, output can be either 0 or 1
+  // TODO: refactor with default parameter to handle n = undefined
+
+  return Math.floor(n * Math.random());
+}
+
+function weatherOnDemand(args) {
```

Comment:
I think that I'd rather refactor than write if reasonable. Could we have an ES5 implementation in here that we could refactor?